### DISCUSSION TITLE
ROS2 Router: Service Call Request made, but no Response

### DISCUSSION THREAD

**Discussion by Thongtdextra** [eProsima/Fast-DDS#5167] [Q&A]
I have 2 host in 1 LAN, 1 Domain. HostA have interface 20.13.21.128 while HostB have interface 20.13.21.24. One of 1 host is local_host_only.

In theory, if you want 2 host with differ domain can communicate each other then you have to setup ddsrouter in both host.
I done with setup and 2 host can see topic each other but i have some problem with service.
In hostB, i run a server service, it can operated in local (if you run ros2 service call in local host then you can receive respone)
but if you run this command: ros2 service call in then hostB can take a request from hostA-client service but hostA can not receive a respone from hostB ?

i think in local participant evething work fine but stuck in wan participant. 

u have any ideal for this case ? structure of service is 
![image](https://github.com/user-attachments/assets/27333bf1-04d4-4170-ab7e-1762a97395f3) and here is my test: 
![image](https://github.com/user-attachments/assets/f07b5559-b6b9-43df-b433-26ecbba420d2)

**Comment by JesusPoderoso** [comment]
Hi @Thongtdextra, thanks for your report.
I am closing this ticket in favor of 
- https://github.com/eProsima/DDS-Router/discussions/464

We will review it in the following days and get back to you with some feedback.
